addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -763,6 +763,76 @@ sitemap modbus_ex_scaling label="modbus_ex_scaling"
 
 See [transformation example](#transformation-example-scaling) for the `divide10.js` and `multiply10.js`.
 
+### Dimmer Example
+
+_(thanks to [rossko57](https://community.openhab.org/t/modbus-openhab2-binding-available-for-alpha-testing/27657/560?u=ssalonen)!)_
```

Comment:
An `Also-by:` line for rossko57 in the commit message would be a better way to keep track of contributions (see [Sign your work](https://www.openhab.org/docs/developer/contributing/contributing.html#sign-your-work)).

Reply:
Good point! I reworded the commit messages to include
```
Also-by: <rossko@culzean.clara.co.uk> (github: Rossko57)
```

Reply:
Can you then remove this line?

[Sign your work](https://www.openhab.org/docs/developer/contributing/contributing.html#sign-your-work):
> using your real name (sorry, no pseudonyms or anonymous contributions.)

Are you OK with adding your real name to the sign off @Rossko57 ?

Reply:
Rossko57's real name now used in commit message

Reply:
Good! So please remove the "thanks to" line because we will thank @Rossko57 forever using the sign off line in the commit. :grinning:

Reply:
Done.